src/main/java/tech/pegasys/pantheon/plugin/services/storage/KeyValueStorage.java

```diff
@@ -23,6 +23,8 @@
  *
  * <p>Behaviour expected with regard to key to value mapping is that of a map, one key maps to one
  * value, when a new value is added with an existing key, that key now points at the new value.
+ *
+ * <p>It is expected that each entry in the keyset is mapped to a non-null value.
```

Comment:
```suggestion
 * <p>All keys and values must be non-null.
```
Which matches the style ConcurrentHashMap uses and covers both keys and values:
`<p>All arguments to all task methods must be non-null.`